src/components/PageCounter/PageCounter.tsx

```diff
@@ -39,8 +36,12 @@ export const PageCounter: React.FC<Props & ElementProps> = ({
     [decorators?.unitForTotal],
   )
   const unitText = useMemo(() => executeDecorator(UNIT_TEXT, decorators?.unit), [decorators?.unit])
-  const rangeSeparator = decorators?.rangeSeparator ?? RANGE_SEPARATOR
-
+  const rangeSeparatorDecorators = useMemo(
+    () => ({
+      visuallyHiddenText: () => executeDecorator(RANGE_SEPARATOR, decorators?.rangeSeparator),
+    }),
+    [decorators?.rangeSeparator],
+  )
```

Comment:
既存のまま `–` ダーシをデフォルトにしたいです。

```suggestion
  const rangeSeparatorDecorators = useMemo(
    () => ({
      text: () => executeDecorator(RANGE_SEPARATOR_TEXT, decorators?.rangeSeparator),
      visuallyHiddenText: () => executeDecorator(RANGE_SEPARATOR_VISUALLY_HIDDEN_TEXT, decorators?.rangeSeparator),
    }),
    [decorators?.rangeSeparator],
  )
```

Reply:
↑だと記号がvisuallyHiddenにも設定されてしまうので駄目っぽいですね。
別途対応します